CHANGELOG.md

```diff
@@ -11,28 +11,29 @@
 
 - Introduced abstraction for getting migrations to make migration bundling easier #2775
 - Allow timestamp with timezone on mssql databases #2724
-- Feature/2690: Multiple migration directories #2735
+- Allow specifying multiple migration directories #2735
 - Allow cloning query builder with .userParams({}) assigned to it #2802
+- Allow chaining of increment, decrement, and update #2740
 - Kill queries after timeout for PostgreSQL #2636
 - Allow table names with `forUpdate`/`forShare` #2834
-- Added `onVal` and the associated `not` / `and` / `or` methods for using values in `on` clauses within joins.
```

Comment:
Judging by https://github.com/tgriesser/knex/commit/ca00ddb9c546ca47240cc9644a7a2ec5f18445d0 this was actually reverted and I don't see any commit where it would have been restored.

Reply:
Actually, I'm wrong, that was a different thing. Restored entry.